crates/bevy_internal/src/default_plugins.rs

```diff
@@ -133,10 +133,50 @@ impl PluginGroup for DefaultPlugins {
             group = group.add(bevy_gizmos::GizmoPlugin);
         }
 
+        group = group.add(IgnoreAmbiguitiesPlugin);
+
         group
     }
 }
 
+struct IgnoreAmbiguitiesPlugin;
+
+impl Plugin for IgnoreAmbiguitiesPlugin {
+    fn build(&self, app: &mut bevy_app::App) {
+        // bevy_ui owns the Transform and cannot be animated
+        #[cfg(all(feature = "bevy_animation", feature = "bevy_ui"))]
+        app.ignore_ambiguity(
+            bevy_app::PostUpdate,
+            bevy_animation::animation_player,
+            bevy_ui::ui_layout_system,
+        );
+
+        if let Ok(render_app) = app.get_sub_app_mut(bevy_render::RenderApp) {
```

Comment:
```suggestion
        #[cfg(feature = "bevy_render")]
        if let Ok(render_app) = app.get_sub_app_mut(bevy_render::RenderApp) {
```
not sure that works directly on an if or if you'll need to add a scope